podvm-mkosi/mkosi.skeleton/usr/lib/systemd/system/kata-agent.service.d/10-override.conf

```diff
@@ -0,0 +1,10 @@
+# On a read-only fs the kata-agent config is created in /run/peerpod, since it contains
+# a parameter that can be set at pod creation time.
+[Unit]
+ConditionKernelCommandLine=
+
+[Service]
+ExecStart=
+ExecStart=/usr/local/bin/kata-agent --config /run/peerpod/agent-config.toml
+ExecStop=
+ExecStopPost=/usr/local/bin/kata-agent-clean --config /run/peerpod/agent-config.toml
```

Comment:
Override for process-user-data service is missing.

Reply:
indeed. thx